pizzeria-backend/Models/OrderedProduct.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace pizzeria_backend.Models
+{
+    public class OrderedProduct
+    {
+        public int Id { get; set; }
+        [ForeignKey("Product")]
+        public int ProductId { get; set; }
+
+        [JsonIgnore]
+        public Product Product { get; set; }
+        [ForeignKey("Order")]
```

Comment:
Leave blank lines between annotations and fields

Reply:
Done